gulpfile.js

```diff
@@ -89,6 +89,10 @@
 
     gulp.task(
         'package:clean',
+        [
+            'qa',
```

Comment:
is `clean` really dependant on `qa` and `build`. Often you want to just be able to start fresh so a `clean` to wipe everything can help. I know for this particular case the tests etc are only run using the unbuilt packages.